modules/phase_field/tests/initial_conditions/MultiSmoothSuperellipsoidIC_Test.i

```diff
@@ -0,0 +1,97 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 22
+  ny = 22
+  nz = 22
+  xmax = 100
+  ymax = 100
+  zmax = 100
+  elem_type = HEX8
+[]
+
+[Variables]
+  [./c]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[ICs]
+  [./c]
+    type = MultiSmoothSuperellipsoidIC
+    variable = c
+    invalue = 1
+    outvalue = 0.1
+    bubspac = 20
+    numbub = 2
+    semiaxis_b_variation = 0.15
+    semiaxis_variation_type = uniform
+    semiaxis_a_variation = 0.2
+    semiaxis_a = 5
+    semiaxis_b = 17
+    semiaxis_c = 5
+  [../]
+[]
+
+[Kernels]
+  [./ie_c]
+    type = TimeDerivative
+    variable = c
+  [../]
+  [./diff]
+    type = MatDiffusion
+    variable = c
+    D_name = D_v
+  [../]
+[]
+
+[BCs]
+  [./Periodic]
+    [./all]
+      auto_direction = 'x y z'
+    [../]
+  [../]
+[]
+
+[Materials]
+  [./Dv]
+    type = GenericConstantMaterial
+    prop_names = D_v
+    prop_values = 0.074802
+    block = 0
+  [../]
+[]
+
+[Postprocessors]
+  [./bubbles]
+    type = FeatureFloodCount
+    variable = c
+    execute_on = 'initial timestep_end'
+  [../]
+[]
+
+[Executioner]
+  # Preconditioned JFNK (default)
+  type = Transient
+  scheme = bdf2
+  solve_type = PJFNK
+  petsc_options_iname = '-pc_type -pc_hypre_type -ksp_gmres_restart -mat_mffd_type'
+  petsc_options_value = 'hypre boomeramg 101 ds'
+  l_max_its = 20
+  l_tol = 1e-4
+  nl_max_its = 20
+  nl_rel_tol = 1e-9
+  nl_abs_tol = 1e-11
+  start_time = 0.0
+  num_steps = 1
+  dt = 100.0
+  [./Adaptivity]
+    refine_fraction = .5
+  [../]
```

Comment:
This is a pretty hefty test! My guess is that this doesn't run in under 2 seconds or even in that ballpark. How about in full debug mode on Linux? I guess we'll see soon enough.